ill-fated-ducks/workforce-management
# Az fix

## Status
Choose:
**READY**


## Description
So I removed the project and cloned again, removed migrations, views and scaffolded again, after fixing some methods on seedData, it is populating the tables without exceptions. It seems it is working fine.
**Checkout to my branch and just try to build it but make sure your tables are empty** 

## Todos

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.
1. git fetch
2. git checkout az-fix
3.  clean up the tables in your db
4.  build it 
5. It should populate all the tables

## Impacted Areas in Application
Everything 

## Link to Feature Ticket
#34 #33 #31 
@ill-fated-ducks
